mitreattack/navlayers/layerGenerator_cli.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+
+try:
+    from .generators.overview_generator import OverviewGenerator
+    from .generators.usage_generator import UsageGenerator
+except ImportError:
+    from mitreattack.navlayers.generators.overview_generator import OverviewGenerator
+    from mitreattack.navlayers.generators.usage_generator import UsageGenerator
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Generate an ATT&CK Navigator layer')
+    parser.add_argument('-m', '--mode', choices=['overview', 'usage'], required=True,
+                        help='Generate a complete overview or the specific usage for a entity')
+    parser.add_argument('-t', '--type', choices=['group', 'software', 'mitigation'], required=True,
+                        help='What kind of object to build the layer for')
+    parser.add_argument('-i', '--identity', help='The identity (id/name) of the desired entity '
+                                                 '(required for usage generation)', default=None)
+    parser.add_argument('-o', '--output', help='Path to the output layer file', default='generated_layer.json')
+    parser.add_argument('--matrix', help='Which matrix to build off of', default='enterprise')
```

Comment:
This should be `domain`, not matrix.

Reply:
That would be more accurate, I'll make the change.